src/styles/blocks/card/blockscard/card.scss

```diff
@@ -0,0 +1,45 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-card;
+  border-radius: 5px;
+  transition: 300ms linear;
```

Comment:
It's a good practice to specify which properties should be animated in the transition. For example, if you are only changing the transform property, you should specify it like this: transition: transform 300ms linear. This way, the browser won't have to guess which properties to animate, which can lead to better performance.